content/docs/command-reference/plots/diff.md

```diff
@@ -6,76 +6,79 @@ plotting them in a single image.
 
 `revisions` are Git commit hashes, tag, or branch names. If none are specified,
-`dvc plots diff` compares metrics currently present in the
-<abbr>workspace</abbr> (uncommitted changes) with the latest committed version.
-A single specified revision results in plotting the difference in metrics
-between the workspace and that version.
+`dvc plots diff` compares targets currently present in the
+<abbr>workspace</abbr> (uncommitted changes) with their latest committed
+versions (required). A single specified revision results in plotting the
+difference between the workspace and that version.
 
-In contrast to commands such as `git diff`, `dvc metrics diff` and
-`dvc params diff`, **any number of `revisions` can be provided**, and the
+In contrast to commands such as `git diff`, `dvc metrics diff`, and
+`dvc params diff`, **any number of revisions can be provided**, and the
 resulting plot shows all of them in a single output.
 
-This command can work with metric files that are committed to a repository
-history, data files controlled by DVC, or any other file in the workspace. In
-the case of DVC-tracked `datafile`, the `revisions` are used to find the
-corresponding [DVC-files](/doc/user-guide/dvc-file-format).
+The plot style can be customized with
+[plot templates](/doc/command-reference/plots#plot-templates), using the
+`--template` option. To learn more about metric file formats and templates
+please see `dvc plots`.
+
+> Note that the default behavior of this command can be modified per metrics
+> file with `dvc plots modify`.
 
 ## Options
 
-- `-d [DATAFILE], --datafile [DATAFILE]` - metrics file to visualize.
+- `--targets <path>` (**required**) - metrics file to visualize.
+
+- `-o <path>, --out <path>` - name of the generated file. By default, the output
+  file name is equal to the input filename with a `.html` file extension (or
+  `.json` when using `--show-vega`).
 
-- `-t [TEMPLATE], --template [TEMPLATE]` -
+- `-t <name_or_path>, --template <name_or_path>` -
   [plot template](/doc/command-reference/plots#plot-templates) to be injected
   with data. The default template is `.dvc/plots/default.json`. See more details
   in `dvc plots`.
 
-- `-f FILE, --file FILE` - name of the generated file. By default, the output
-  file name is equal to the input filename with additional `.html` suffix or
-  `.json` suffix for `--no-html` mode.
-
-- `--no-html` - do not wrap output Vega specification (JSON) with HTML.
-
-- `-x X` - field name for X axis. An auto-generated `index` field is used by
-  default.
+- `-x <field>` - field name from which the X axis data comes from. An
+  auto-generated `index` field is used by default. See
+  [Custom templates](/doc/command-reference/plots#custom-templates) for more
+  information on this `index` field.
 
-- `-y Y` - field name for Y axis. The last column or field found in the
-  `datafile` is used by default.
+- `-y <field>` - field name from which the Y axis data comes from. The last
```

Comment:
should we mention that it can be a number as well?

Reply:
Sure. Done in c0ce897b